sdk/servicebus/Azure.Messaging.ServiceBus/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,7 @@
+# Baselining these until the next GA as they are not actual compat issues.
+CannotRemoveAttribute : Attribute 'System.ComponentModel.EditorBrowsableAttribute' exists on 'Azure.Messaging.ServiceBus.ServiceBusModelFactory.TopicProperties(System.String, System.Int64, System.Boolean, System.TimeSpan, System.TimeSpan, System.TimeSpan, System.Boolean, Azure.Messaging.ServiceBus.Administration.EntityStatus, System.Boolean)' in the contract but not the implementation.
```

Comment:
These were needed as I removed the EditorBrowsable.Never attribute that was incorrectly added on the factory methods before. 
/cc @weshaggard